src/components/templates/Audience/Audience.scss

```diff
@@ -2,142 +2,149 @@
 
 $light: #ffffff;
 
+// If you change this, make sure to also change it in Audience.tsx's SEAT_SIZE
+$seat-size: 4vh;
+$seat-size-min: 25px;
```

Comment:
[NB] This comment might not be needed anymore since we use the `--seat-size`/etc variables in the component now too. Probably doesn't hurt to leave it here though..